### DISCUSSION TITLE
@powerGridScripts Conflicts with Livewire alpine js

### DISCUSSION THREAD

**Discussion by thedangler** [Power-Components/livewire-powergrid#777] [Q&A] [answered]
Hello,
I'm trying to make a pop up modal with alpinejs following this tutorial https://codecourse.com/watch/global-modals-with-livewire.
In the Modal.blade file I have
"x-data="{ show: @entangle($attributes->wire('model')).defer}" x-show="show" x-on:keydown.escape.window="show = false" in a div tag.
When I run it, the pop up modal opens fine, however when I hit the escape key I get the following error.
"cdn.min.js:5 Uncaught (in promise) TypeError: u is not a function"

When I remove the @powerGridScripts tag the modal works just fine.
I can't seem to figure out how alpinejs is conflicting.

Does't livewire use alpinejs so why is powergrid also adding it with the scripts?

I'm using latest version of livewire and laravel and followed the powergrid install instructions for existing project.

Please advise.

**Comment by luanfreitasdev** [comment, marked as answer]
Hello. This happens because until version 3, Alpine already comes with cdn in the settings. But you can simply change it to 'null'

https://livewire-powergrid.com/get-started/configure.html?id=configure#_2-alpine-js
